Графика для всех страниц

DinARTSky opened the pull request:
---
:mortar_board: [Разметка всех страниц](https://up.htmlacademy.ru/htmlcss/36/user/373113/tasks/5)

:boom: https://htmlacademy-htmlcss.github.io/373113-gllacy-36/2/

---
:mortar_board: [Графика для всех страниц](https://up.htmlacademy.ru/htmlcss/36/user/373113/tasks/7)

DinARTSky committed:
first step

указал DOCTYPE

DinARTSky committed:
разметка header main footer

разметка header main footer

DinARTSky committed:
Разметка страницы

Разметил блоки страницы

DinARTSky committed:
Добавление заголовков

Добавил заголовки в секции

DinARTSky committed:
Разметка продолжение

DinARTSky committed:
Классы в header

Расставил классы в header

DinARTSky committed:
разметка страницы index

разметка страницы index

DinARTSky committed:
каталог, фильтры

Создал файл каталог, разметил фильтры

DinARTSky committed:
Разметка после видео

DinARTSky committed:
Размечена главная страница

Размечена главная страница

DinARTSky committed:
разметка двух старниц

разметка двух старниц

senchkim commented on catalog.html:
```diff
@@ -0,0 +1,342 @@
+<!DOCTYPE html>
```
https://up.htmlacademy.ru/htmlcss/36/criteries#HTML-07
На обоих страницах, поправить ошибки типа `error`

senchkim commented on catalog.html:
```diff
@@ -0,0 +1,342 @@
+        <h1>Сливочное мороженное</h1>
+      </header>
+      
+      <section class="buttericecream-catalog-products">
```
Хороший тон разделять слова через дефис:
`buttericecream` => `butter-ice-cream`

senchkim commented on catalog.html:
```diff
@@ -0,0 +1,342 @@
+        <ul class="product-list">
+          <li class="product-card">
+            <a href="product.html">
+              <img src="product-card-image">
```
у тэга `img` - обязательный атрибут `alt`  👍

senchkim commented on catalog.html:
```diff
@@ -0,0 +1,342 @@
+        </ul>
+      </nav>
+
+      <span>.html academy</span>
```
плохо когда тэги без классов, так как в дальнейшем стилизовать все тэги будем через `КЛАССЫ` 👍

senchkim commented on index.html:
```diff
@@ -3,10 +3,366 @@
+          <li class="slider-item">
+            <h3 class="slider-title">Сливочное мороженое со вкусом банана</h3>
+            <p class="slider-description">Сливочное мороженое с ярким банановым вкусом подарит вам свежесть и наслаждение даже в самый жаркий летний день.</p>
+            <button>Заказать</button>
```
Писал уже все тэги должны быть с классами!

senchkim commented on index.html:
```diff
@@ -3,10 +3,366 @@
+        <p class="subscribe-description">Подпишитесь на нашу сладкую рассылку и будьте всегда в курсе всего самого вкусного, что у нас происходит. Обещаем не спамить и не слать всякой ненужной ерунды. Честно =)
+        </p>
+        <form>
+          <input type="" name="">
```
тут явно нужно доделать(

senchkim commented on index.html:
```diff
@@ -3,10 +3,366 @@
+        </p>
+        <form>
+          <input type="" name="">
+          <input type="submit" name="">
```
тут явно нужно доделать(

senchkim commented on index.html:
```diff
@@ -3,10 +3,366 @@
+        <p class="delivery-description">Хочется полакомиться любимым десертом, но нет времени съездить в магазин? Закажите доставку мороженого на дом, и курьер привезёт вам ваш заказ в течение часа!
+        </p>
+
+        <form>
```
тут явно нужно доделать(

senchkim commented on index.html:
```diff
@@ -3,10 +3,366 @@
+
+      <span>.html academy</span>
+      <p>Сделано в .html academy © 2021</p>
+    </footer>
```
В целом работа хорошая 💯

DinARTSky committed:
Каталог

DinARTSky committed:
Alt, social icon

добавил альты и иконки соцсетей отредактировал

DinARTSky committed:
Исправил замечания по разметке

Исправил замечания по разметке, добавил всем изображениям альты и всем тегам классы

senchkim commented on index.html:
```diff
@@ -3,10 +3,428 @@
   <head>
     <meta charset="utf-8">
     <title>HTML Academy: Глейси</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
```
Ты отправил сразу 2 задания - это плохо так лучше не делать!
Потому что приняв одно задание автоматически принялось и второе(

DinARTSky replied:
Хорошо, принял.
Как теперь быть с заданием по графике?
